mobile/src/components/courses/components/category-list/components/category/category.tsx

```diff
@@ -22,6 +22,7 @@ const Category: FC<Props> = ({ keyName, name, onPress }) => {
   const hasImage = imageKeys.includes(keyName);
 
   const handlePress = (): void => {
+    if (!onPress) return;
     setIsActive(!isActive);
     onPress();
```

Comment:
```suggestion
    setIsActive(!isActive);
    onPress?.();
```

Reply:
But I want without that effect )  setIsActive(!isActive);
If we didn't pass onPress
<img src = "https://user-images.githubusercontent.com/82529236/187077668-54d22185-2174-48aa-9ec4-a49156b084c5.gif" width="35%">

Reply:
Let's leave this for now, we will remove this after pr with categories is merged